Core/include/Acts/Utilities/TrackHelpers.hpp

```diff
@@ -159,14 +160,21 @@ Result<void> smoothTracks(
 template <typename track_proxy_t>
 Result<std::pair<typename track_proxy_t::ConstTrackStateProxy, double>>
 findTrackStateForExtrapolation(
-    const GeometryContext &geoContext, track_proxy_t &track,
+    const GeometryContext &geoContext, const track_proxy_t &track,
```

Comment:
another API change that might count as a breaking change, but again probably won't actually break any real code.

Reply:
I think this one should be fine as something non const will still pass and something const too

Reply:
yeah, I think it only matters if the exact signature needs to match. I assume not here.